addon/services/is-mobile.js

```diff
@@ -10,9 +9,7 @@ import isMobile from 'ismobilejs';
  * agent header is blank.
  */
 export default Service.extend({
-  fastboot: computed(function() {
-    return getOwner(this).lookup('service:fastboot');
-  }),
+  fastboot: service('fastboot'),
```

Comment:
This will break any app that does not use FastBoot. Injecting a non-existent service will cause a runtime exception. Using `getOwner` is the recommended way for addons to access the fastboot service.

Reply:
Thanks for pointing out, will revert back